docs/docsite/rst/dev_guide/testing/sanity/metaclass-boilerplate.rst

```diff
@@ -0,0 +1,23 @@
+Sanity Tests » metaclass boilerplate
```

Comment:
A minor tweak for consistency with the title of the future boilerplate page title:

```suggestion
Sanity Tests » __metaclass__ boilerplate
```

Reply:
Hmm... I forgot that the title doesn't need to match up with the test name.... That being the case I'll change this to __metaclass__ = type boulerplate.

Reply:
Fixed and updated grandfathered list pushed.  If it passes, I'll merge it and let slack know that everything is stale.